hack/restructure_file.py

```diff
@@ -0,0 +1,56 @@
+import json
+import sys
+
+filename= sys.argv[1]
+savefile= sys.argv[2]
+printfile= sys.argv[3]
+
+
+def has_body(list1):
```

Comment:
Can you change `list1` and `list2`, probably something in the lines of `item`.

Also the `message` and `message1` could be called `hasError` or `hasBody`.

None of them convey a clear message.